addons/gamification/models/res_users.py

```diff
@@ -4,6 +4,12 @@
 from odoo import _, api, fields, models
 
 
+class ResUsersLog(models.Model):
+    _inherit = 'res.users.log'
+
+    last_interacted_date = fields.Datetime('Last interacted', default=fields.Datetime.now())
```

Comment:
oups, classic mistake, this will be executed once when the server starts and it will have the same value for all records
```suggestion
    last_interacted_date = fields.Datetime('Last interacted', default=fields.Datetime.now)
```

```
>>> def get_default():
...     print("bouh")
...     return 5
... 
>>> def f(x=get_default()):
...     pass
... 
bouh
```